app/src/client/components/form/TextArea.tsx

```diff
@@ -5,14 +5,16 @@ interface TextAreaProps {
   value: string;
   placeholder: string;
   onChange: (value: string) => void;
+  isDisabled?: boolean;
 }
 
-export const TextArea: React.FC<TextAreaProps> = ({ id, value, placeholder, onChange }) => (
+export const TextArea: React.FC<TextAreaProps> = ({ id, value, placeholder, onChange, isDisabled }) => (
```

Comment:
we could stick to the attribute name disabled instead of isDisabled.  html is using it, so we can use it as well.

Reply:
Sure, make sense. I will push my changes shortly to the same PR